bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/handler/NanoleafControllerHandler.java

```diff
@@ -824,8 +830,8 @@ private void sendRhythmCommand(Command command) throws NanoleafException {
         OpenAPIUtils.sendOpenAPIRequest(setNewRhythmRequest);
     }
 
-    private String getAddress() {
-        return StringUtils.defaultString(this.address);
+    private @Nullable String getAddress() {
+        return address;
```

Comment:
Is this method still needed?

Reply:
No, none of these private getters and setters is needed. But I do not want to refactor the whole binding. I just want to get rid of the dependency.